### ISSUE TITLE
Error on iOS version

### ISSUE THREAD

**Issue by javipacheco** [Abedalkareem/games_services#1]
I'm trying to add the library to my game. In Android works fine, but I have this problem in ios

```
    === BUILD TARGET firebase_crashlytics OF PROJECT Pods WITH CONFIGURATION Debug ===
    /Users/gadestudios/Development/flutter/.pub-cache/hosted/pub.dartlang.org/games_services-0.1.1/ios/Classes/SwiftGamesServicesPlugin.swift:16:43: error: cannot call value of non-function type 'GKLocalPlayer'
        let player = GKLocalPlayer.localPlayer()
                     ~~~~~~~~~~~~~~~~~~~~~~~~~^~
```

I have been following [this article](https://itnext.io/how-to-integrate-gamekit-and-google-play-services-flutter-4d3f4a4a2f77) and I think that all is ok

Could you please help me?

**Comment by Abedalkareem** [comment]
Hi,
I think there is an update on the GameKit, so now instead of `GKLocalPlayer.localPlayer()` it's `GKLocalPlayer.local`.
I'll fix this issue today and update you, Thank you for reporting.

**Comment by Abedalkareem** [comment]
The issue solved in version 0.2.1, thank you for reporting the issue!
